src/components/clusterexplorer/node.resource.ts

```diff
@@ -56,6 +57,24 @@ export class ResourceNode extends ClusterExplorerNodeImpl implements ClusterExpl
     get isExpandable(): boolean {
         return getChildSources(this.kind).length > 0;
     }
+
+    async getPathApi(namespace: string): Promise<string> {
+        const resources = this.kind.pluralDisplayName.replace(/\s/g, '').toLowerCase();
+        const version = await getResourceVersion(resources);
+        const baseUri = (version === 'v1') ? `/api/${version}/` : `/apis/${version}/`;
+        let namespaceUri = '';
+        switch (resources) {
```

Comment:
Same here, what do you mean? The getResourceVersion allows to find info about every resource so that we can set up a watch. Tekton is supported. The new additional resources can be watched correctly. Tested with a running pipeline